.travis/docker_test.sh

```diff
@@ -229,4 +240,4 @@ fi
 # Success
 echo 'Frappe app '${FRAPPE_APP_TO_TEST}' tests finished'
 echo 'Check the CI reports and logs for details.'
-exit 0
+exit 0
```

Comment:
@AminovE99 again, fix your IDE to stop removing end of lines at the end of the file!!
```suggestion
exit 0

```